src/controllers/sessionController.ts

```diff
@@ -67,11 +68,22 @@ export default class SessionController {
         TE.toUnion
       )();
 
+      // Read the assertionRef related to the User for Lollipop.
+      const errorOrMaybeAssertionRef = await this.sessionStorage.getLollipopAssertionRefForUser(
+        user
+      );
+      if (E.isLeft(errorOrMaybeAssertionRef)) {
+        return ResponseErrorInternal(
+          `Error retrieving the assertionRef: ${errorOrMaybeAssertionRef.left.message}`
+        );
+      }
+
       if (UserV5.is(user)) {
         // All required tokens are present on the current session, no update is required
-        return ResponseSuccessJson({
+        return ResponseSuccessJson<PublicSession>({
```

Comment:
it's not necessary
```suggestion
        return ResponseSuccessJson({
```

Reply:
Fixed in [9ce3eea](https://github.com/pagopa/io-backend/pull/993/commits/9ce3eea19c71a4b21bbfc1a2b91ad73b1a939808)